groups/sig-k8s-infra/groups.yaml

```diff
@@ -23,11 +23,11 @@ groups:
       - thockin@google.com
       - linusa@google.com
 
-  # TODO(spiffxp): rename this to wg-k8s-infra-private at some point
-  - email-id: k8s-infra-team-private@kubernetes.io
-    name: k8s-infra-team-private
+  - email-id: sig-k8s-infra-private@kubernetes.io
```

Comment:
Some cleanup needed ?

Reply:
SG, I'll look at pruning members from -private in a separate PR